packages/cashier/src/pages/deposit/crypto-deposit/__tests__/crypto-deposit.spec.tsx

```diff
@@ -437,4 +455,37 @@ describe('<CryptoDeposit />', () => {
 
         expect(screen.getByText('RecentTransactions')).toBeInTheDocument();
     });
+
+    it('should show AlertBanner for minimum deposit when third-party payment processor is used (CoinsPaid)', () => {
+        const minimum_deposit = 2;
+        const currency = 'tUSDT';
+        const mockRootStore = {
+            client: {
+                currency: 'tUSDT',
+            },
+            modules: {
+                cashier: {
+                    onramp: {
+                        is_deposit_address_loading: false,
+                        api_error: '',
+                        deposit_address: 'tb1ql7w62elx9ucw4pj5lgw4l028hmuw80sndtntxt',
+                        pollApiForDepositAddress: jest.fn(),
+                    },
+                    transaction_history: {
+                        crypto_transactions: [{}],
+                        onMount: jest.fn(),
+                    },
+                    general_store: {
+                        setIsDeposit: jest.fn(),
+                    },
+                },
+            },
+        };
+        renderWithRouter(<CryptoDeposit />, mockRootStore as TRootStore);
+        expect(
+            screen.getByText(
+                `A minimum deposit value of ${minimum_deposit} ${currency} is required. Otherwise, the funds will be lost and cannot be recovered.`
```

Comment:
```suggestion
                `A minimum deposit value of ${minimum_deposit} ${mock.client.currency} is required. Otherwise, the funds will be lost and cannot be recovered.`
```